### ISSUE TITLE
Add firejail.config option to prevent loading of user profiles.

### ISSUE THREAD

**Issue by crass** [netblue30/firejail#2155]
On systems where the unprivileged user does not have access to root, the sysadmin should have the ability to disallow custom profiles.  This will lower the attack surface on firejail itself.  In order for this to be effective, when user profiles are disabled, the `--profile` option should only allow loading files from `${CFG}`.  What should this option be named in `firejail.config`?  Perhaps `user_profiles`? And I suggest having them off by default (secure default).

**Comment by smitsohu** [comment]
> This will lower the attack surface on firejail itself.

There is still always the command line, though.

**Comment by reinerh** [comment]
Right, when the users can still override everything on the command line, but no longer in their config files, that's an even bigger reason to keep it on by default.

**Comment by crass** [comment]
@smitsohu very good point... :frowning_face: As I see it now, either we should not do this or it needs to disable any options which can loosen the container.  Options that can tighten up the container should be ok.  This will require a more thorough evaluation of the options...

**Comment by crass** [comment]
So I'm thinking that the pull request shouldn't be merged as-is because it gives the illusion of locking down the users.  I was thinking of abandoning the rest of the work because it seemed like a PITA.  But, it looks like it could be easily implemented in `check_arg` using `checkcfg` and a list of options we want to disallow.

So now the PITA part is getting the list of options that we should disallow.  If someone or the collective wants to come up with such a list, I could implement it.  I'm not interested in doing that part myself though.

**Comment by chiraag-nataraj** [comment]
Are we interested in moving forward with this or should we close this?
